### PR TITLE
docs: Update How to send data from DataDog agent

### PR DESCRIPTION
**Pull request by YuriKravetc**
Providing additional description, pictures and changing structure.

### CODE REVIEW

File: docs/Single-server-VictoriaMetrics.md [hunk 1]
```diff
@@ -343,20 +343,59 @@ See also [vmagent](https://docs.victoriametrics.com/vmagent.html), which can be
...
+DataDog agent allows configuring destinations for metrics sending via ENV variable `DD_DD_URL` or via [configuration file](https://docs.datadoghq.com/agent/guide/agent-configuration-files/) in section `dd_url`.
 
-### Cluster version of VictoriaMetrics:
+<p align="center">
+  <img src="Sending_DD_metrics_to_VM.png" width="800">
+</p>
+
+Depending on where you want to send metrics, the URL for VictoriaMetrics will be different:
+1. Run DataDog using ENV variable `DD_DD_URL=http://victoriametrics-host:8428/datadog` for sending metrics to single-node VictoriaMetrics. Alternatively add `dd_url: http://victoriametrics-host:8428/datadog` to DataDog yaml configuration file.
+2. Run DataDog using variable `DD_DD_URL=http://vminsert-host:8480/insert/0/datadog` for sending metrics to the vminsert component of the cluster version of VictoriaMetrics. Alternatively add `dd_url: http://vminsert-host:8480/insert/0/datadog` to DataDog yaml configuration file.
```
Comment (hagen1778) [inline docs/Single-server-VictoriaMetrics.md:356]: wdyt about adding datadog endpoints for single/cluster to https://docs.victoriametrics.com/url-examples.html
and just refer them here?

File: docs/Single-server-VictoriaMetrics.md [hunk 2]
```diff
@@ -343,20 +343,59 @@ See also [vmagent](https://docs.victoriametrics.com/vmagent.html), which can be
...
-VictoriaMetrics doesn't check `DD_API_KEY` param, so it can be set to arbitrary value.
+### Sending metrics to Datadog and VictoriaMetrics
+ 
+DataDog allows configuring [Dual Shipping](https://docs.datadoghq.com/agent/guide/dual-shipping/) for metrics sending via ENV variable `DD_ADDITIONAL_ENDPOINTS` or via configuration file `additional_endpoints` described in section YAML configuration.
+ 
+<p align="center">
+  <img src="Sending_DD_metrics_to_VM_and_DDhq.png" width="800">
+</p>
+ 
+1. Run DataDog using ENV variable `DD_ADDITIONAL_ENDPOINTS='{\"http://victoriametrics:8428/datadog\": [\"yourAPIkey\"]}'` variable for sending metrics to the single-node VictoriaMetrics on top of DataDoghq.
```
Comment (hagen1778) [inline docs/Single-server-VictoriaMetrics.md:368]: are you sure `yourAPIkey` is needed for sending data to VM?

File: docs/Single-server-VictoriaMetrics.md [hunk 3]
```diff
@@ -453,6 +495,8 @@ incoming data with the added tags to a centralized VictoriaMetrics specified via
 
 See [these docs](https://docs.victoriametrics.com/vmagent.html#adding-labels-to-metrics) for details on how to add labels to metrics at `vmagent`.
 
+VictoriaMetrics doesn't check `DD_API_KEY` param, so APIkey can be set to arbitrary value.
```
Comment (hagen1778) [inline docs/Single-server-VictoriaMetrics.md:498]: Can we not set it at all?

**Review by hagen1778 — CHANGES_REQUESTED** [review]

File: docs/Single-server-VictoriaMetrics.md (hunk 3 above)
Comment (YuriKravetc) [reply docs/Single-server-VictoriaMetrics.md]: Yes, removed.

File: docs/Single-server-VictoriaMetrics.md (hunk 1 above)
Comment (YuriKravetc) [reply docs/Single-server-VictoriaMetrics.md]: Good point
Done. Please check and let me know wdyt.

**Review by hagen1778 — APPROVED** [review]

**Review by hagen1778 — APPROVED** [review]
Thanks!

**Pull request merged**